tests/libtest/test_elasticsearch.py

```diff
@@ -113,14 +127,30 @@ def smallfile_run(self, es):
 
     def test_elasticsearch(self):
         """
-        This test only deploy the elasticsearch module, connect to it with and
-        without credentials and teardown the environment
+        This test do the following operations:
 
-        Args:
-            es (fixture) : fixture that deploy / teardown the elasticsearch
+            * deploy the elasticsearch module
+            * connect to it
+            * run a simple SmallFile benchmark (to verify usability)
+            * dump the results to a file
+            * push the results from the file to the Dev. ES.
+            * teardown the environment
 
         """
 
+        log.info("Test with 'Dummy' Storageclass")
```

Comment:
why is this needed?

Reply:
to verify that the cleanup incase of pvc not bounded is working, and for that i am using "not exist (dummy)" storageclass.